packages/core/src/Stores/traders-hub-store.js

```diff
@@ -297,6 +298,17 @@ export default class TradersHubStore extends BaseStore {
         return eu_related.includes(this.content_flag);
     }
 
+    get restricted_countries_filter_content() {
```

Comment:
@aizad-deriv Would you please also add the type in `packages/stores/types.ts` 🙏🏻